gpt4all-chat/qml/ChatDrawer.qml

```diff
@@ -155,8 +155,8 @@ Drawer {
                                 chatName.readOnly = false
                                 chatName.selectByMouse = true
                             }
-                            Accessible.name: qsTr("Edit the chat name")
-                            Accessible.description: qsTr("Provides a button to edit the chat name")
+                            Accessible.name: qsTr("Edit chat name")
+                            Accessible.description: qsTr("")
```

Comment:
In the case of empty descriptions like this, I think we could remove the whole line that sets it.

Reply:
@manyoso What do you think?

Reply:
Either way is fine with me. I just left them empty in case someone decides to add those descriptions in the future (I doubt it though).
Let me know and I'd be happy to remove them.